### DISCUSSION TITLE
I'm developing a library development template using vue-demi!

### DISCUSSION THREAD

**Discussion by takuma-ru** [vueuse/vue-demi#200] [General]
Hi everyone!

I'm currently creating a template repository for library development using vue-demi.
#### Functions currently implemented
- [x] Library development projects using vite
- [x] Project for testing with vue3(vite)
- [x] Project for testing with vue2(nuxt)

#### The following functions are planned to be implemented in the future
- [ ] Template project for library documentation (nuxt3)
- [ ] Enable development with `tsx
- [ ] Automatic update of test projects
- [ ] Introduction of storybook
- [ ] Introduction of jest (in test project)

It is still in the process of being created, but I encourage everyone to try this template!

I welcome anyone who is willing to help us develop it.

**Comment by thiswallz** [comment]
good luck with that, would be nice if u add some images.
